cli/internal/packager/create.go

```diff
@@ -70,13 +70,18 @@ func addLocalAssets(tempPath componentPaths, assets config.ZarfComponent) {
 	if len(assets.Charts) > 0 {
 		logrus.Info("Loading static helm charts")
 		utils.CreateDirectory(tempPath.charts, 0700)
+		utils.CreateDirectory(tempPath.values, 0700)
 		for _, chart := range assets.Charts {
 			isGitURL, _ := regexp.MatchString("\\.git$", chart.Url)
 			if isGitURL {
 				helm.DownloadChartFromGit(chart, tempPath.charts)
 			} else {
 				helm.DownloadPublishedChart(chart, tempPath.charts)
 			}
+			if chart.Values != "" {
```

Comment:
Yeah my only concern is that we need to decide on naming conventions everywhere for the yaml structure (though we can make them different it might confuse things for us more)

Reply:
updated in the last commit to valuesFile.